src/server.py

```diff
@@ -506,19 +480,17 @@ def run_jobnames_query():
 @json_response
 def run_dailyjob_query():
     start_date, end_date = clean_date_params(request.args)
-    db = create_db_connnection()
-    cursor = db.cursor()
-    query = "select date, platform, branch, numpushes, numjobs, sumduration from dailyjobs \
-             where date>='%s' and date <='%s'\
-             order by case platform \
-                when 'linux' then 1 \
-                when 'osx' then 2  \
-                when 'win' then 3  \
-                when 'android' then 4 \
-                end" % (start_date, end_date)
-    cursor.execute(query)
+
+    data = session.query(Dailyjobs.date, Dailyjobs.platform, Dailyjobs.branch, Dailyjobs.numjobs,
+                         Dailyjobs.sumduration).\
+        filter(Dailyjobs.date.between(start_date, end_date)).order_by(case([
```

Comment:
here we use date between when previous queries used >= and <=, I really like date.between, and would like to use it wherever possible (even if we have to adjust the dates slightly).  Are you sure the dates are correct here, and if so, please consider doing date.between everywhere else.